.travis.yml

```diff
@@ -11,17 +10,9 @@ matrix:
   allow_failures:
     - rust: nightly
 
-env:
-  global:
-    - CLIPPY_RUST_VERSION=nightly-2018-07-24
-
 before_script:
-  - bash -c 'if [[ "$TRAVIS_RUST_VERSION" == "$CLIPPY_RUST_VERSION" ]]; then
```

Comment:
I usually keep this pattern to avoid breaking master when new clippy lints are introduced. Would it be fine for you to keep a pinned toolchain for clippy?